src/drivers/driver.ts

```diff
@@ -634,6 +639,15 @@ export abstract class CMakeDriver implements vscode.Disposable {
         return -1;
       }
 
+      // _beforeConfigureOrBuild contains the only scenario when we would want partial features set
```

Comment:
If this is the case, then should the call to enable the full feature set go there instead? This would also give you the opportunity to turn it off if the conditions started to fail for some reason.